pages/linux/opkg.md

```diff
@@ -0,0 +1,35 @@
+# opkg
+
+> The opkg utility is a lightweight package manager used to install OpenWrt packages.
+
+- Install a package:
+
+`opkg install {{wget}}`
+
+- Remove a package:
+
+`opkg remove {{wget}}`
+
+- Update the list of available packages:
+
+`opkg update`
+
+- Upgrade installed packages:
+
+`opkg upgrade`
```

Comment:
Isn't there a way to upgrade a single package ?

Reply:
Yes there is. `opkg upgrade {{package(s)}}` (where packages are a space-separated list).